project2/styles.css

```diff
@@ -0,0 +1,238 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    background-color: #508d8b;
+    color: black;
+    font-family: termina, sans-serif;
+}
+
+a {
+    text-decoration: none;
+}
+
+
+h2,
+h3,
+h4,
+p {
+    padding-left: 10px;
+}
+
+
+h2,
+h3,
+p {
+    padding-right: 10px;
+}
+
+
+h2 {
+    color: #ae274d;
+    font-family: termina, sans-serif;
+    font-size: 10.5px;
+    font-style: normal;
```

Comment:
The default `font-style` for a heading is already normal, so you can drop this line.
```suggestion
```